frontend/src/components/Body/Tarifs/Tariffs.js

```diff
@@ -127,71 +153,68 @@ function Tariffs() {
-              display: 'flex',
-				  justifyContent: 'center',
-				  flexWrap: 'wrap'
-            }}
-			marginTop={0}
-      >
-        {TariffsData.map((item, index) => (
-          <Grid 
-            item 
-            key={item.id} 
-            xs={12} 
-            sm={6} 
-            md={4} 
-            sx={{ 
-              display: 'flex',
-				  justifyContent: 'center',
-            }}
-          >
-            <Card 
-              sx={{ 
+		<Box
+		paddingTop={{ lg: "50px", md: '50px', sm: '30px', xs: "30px"}}
+		display={'flex'}
+		flexDirection={'row'}
+		gap={{lg: "50px", md: '24px', sm: '24px', xs: "24px"}}
+		flexWrap={'wrap'}
+		justifyContent={'center'}
+		>
+			{TariffsData.map((item, index) => (
+				<Card
+				 sx={{
 					display: 'flex',
-				  justifyContent: 'center',
-                border: '1px solid', 
-                borderColor: (theme) => theme.palette.primary.borderBlue, 
-                borderRadius: '15px', 
-                maxWidth: '376px', 
-					 minWidth: '30%',
-                minHeight: '566px',
-                background: 'transparent',
-                padding: '30px',
-                flexDirection: 'column',
-					 gap: 4,
-				    width: '100%',
-              }}
-            >
-              <Typography
+					justifyContent: 'center',
+					border: '1px solid',
+					borderColor: (theme) => theme.palette.primary.borderBlue, 
+               borderRadius: '15px', 
+					// minHeight: '100%',
+					maxWidth: {
+						xl: '100%',
+						lg: "376px",
+						md: "376px",
+						sm: "376px",
+						xs: "376px",
+					}, 
+					minWidth: {
+						lg: "376px",
+						md: "376px",
+						sm: "376px",
+						xs: "300px",
+					},
+					background: 'transparent',
+               padding: '30px',
+               flexDirection: 'column',
+					// width: '100%'
+					gap: '30px'
+				 }}
+				>
+					<Typography
                 fontFamily={"'Unbounded', sans-serif"}
                 fontWeight={'400'}
-                fontSize={{ md: "30px", sm: '26px', xs: '24px'}}
+                fontSize={{ xl: "38px", lg: "30px", md: "30px", sm: '26px', xs: '24px'}}
                 color={(theme) => theme.palette.primary.borderBlue}
                 variant="h1"
                 display={'center'}
                 justifyContent={'center'}
               >
                 {item.title}
               </Typography>
-              {index === 1 || index === 2 ? (
-                <>
-                  <Typography
+				  { (index === 1 || index === 2) && (
+					<>
+					<Typography
                     variant="h3"
                     fontFamily={"Inter, sans-serif"}
                     fontWeight={'400'}
-                    fontSize={"18px"}
+                    fontSize={{ xl: "24px", lg: "18px", md: "18px", sm: '18px', xs: '18px'}}
                     color={(theme) => theme.palette.primary.main}
                     display={'center'}
                     justifyContent={'center'}
                   >
```

Comment:
Move this styles to a "sx" props, add a "key" props to same components if it locate in a "map" cicle